packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-styles.ts

```diff
@@ -64,32 +82,50 @@ export const getComponentCss = (
         display: 'block',
         height: '100%',
         width: '100%',
-        padding: `${pxToRemWithUnit(12)} ${ITEM_PADDING}`,
+        padding: `13px ${ITEM_PADDING}`,
         margin: 0,
-        border: 0,
+        border: `${borderWidthBase} solid ${borderColor}`,
+        transition: getTransition('border-color'),
+        outline: 0,
         background: backgroundColor,
         color: buttonColor,
         ...textSmallStyle,
-        ...getFocusJssStyle({ color: primaryColor }),
+        overflowWrap: 'normal',
+        position: 'relative',
+        borderRadius: borderRadiusSmall,
+        '&::before': {
+          content: '""',
+          position: 'absolute',
+          ...getInsetJssStyle(-5),
+          border: `${borderWidthBase} solid transparent`,
+          borderRadius: '7px',
```

Comment:
I think, if you would use `boxSizing: 'content-box'`, you could use inset: -6px and use borderRadiusMedium instead of custom 7px definition. See `checkbox-radio-styles.ts`

Reply:
@stephanschroeter If I make `boxSizing: 'content-box'`, the element is getting too big, because it has padding (in comparison to checkbox-wrapper where we don't have padding)
The same behavior you can see here https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing (if you select the second option)